Sources/EpoxyCore/SwiftUI/EpoxySwiftUIHostingView.swift

```diff
@@ -246,6 +241,12 @@ public final class EpoxySwiftUIHostingView<RootView: View>: UIView, EpoxyableVie
     }
   }
 
+  public func forceLayout() {
```

Comment:
could we add doc comments to this function?

Reply:
Thanks for the suggestion. Added the usage description.